web/packages/teleterm/csp.ts

```diff
@@ -22,21 +22,22 @@ export function getConnectCsp(development: boolean) {
     ? 'https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod'
     : 'https://usage.teleport.dev';
 
-  let csp = `
+  const scriptEval = development
+    ? // Required to make source maps work in dev mode.
+      "'unsafe-eval' 'unsafe-inline'"
+    : // Enables WASM initialization with a safer alternative to 'unsafe-eval'.
+      // This source expression applies specifically to WASM.
+      "'wasm-unsafe-eval'";
+
+  return `
 default-src 'self';
 connect-src 'self' ${feedbackAddress};
 style-src 'self' 'unsafe-inline';
 img-src 'self' data: blob:;
 object-src 'none';
 font-src 'self' data:;
+script-src 'self' ${scriptEval};
```

Comment:
Do we need `'self'` though in the packaged app?

Reply:
I was going off the fact that it wasn't previously present, but in that scenario I believe it was covered by `default-src 'self'`. I wonder if it's needed though since technically we read the script from the disk in the packaged app, no?